components/omega/test/CMakeLists.txt

```diff
@@ -146,14 +145,50 @@ target_link_options(
   ${OMEGA_LINK_OPTIONS}
 )
 
-target_link_libraries(${_TestDecompName} ${OMEGA_LIB_NAME} yakl metis)
+target_link_libraries(${_TestDecompName} ${OMEGA_LIB_NAME} spdlog yakl metis pioc)
 
 add_test(
   NAME DECOMP_TEST
   COMMAND ${MPI_EXEC} -n 8 -- ./${_TestDecompName}
 )
 
 
+#############
+# IO test
+#############
+
+set(_TestIOName testIO.exe)
+
+add_executable(${_TestIOName} base/IOTest.cpp)
+
+target_include_directories(
+   ${_TestIOName}
+  PRIVATE
+  ${OMEGA_SOURCE_DIR}/src/base
+  ${OMEGA_SOURCE_DIR}/src/infra
+  ${PARMETIS_ROOT}/include
+)
+
+target_compile_options(
+   ${_TestIOName}
+  PRIVATE
+  ${OMEGA_CXX_FLAGS}
+)
+
+target_link_options(
+   ${_TestIOName}
+  PRIVATE
+  ${OMEGA_LINK_OPTIONS}
+)
+
+target_link_libraries(${_TestIOName} ${OMEGA_LIB_NAME} spdlog yakl metis pioc)
+
+add_test(
+  NAME IO_TEST
+  COMMAND ${MPI_EXEC} -n 8 -- ./${_TestIOName}
+)
+
+
 ##################
 # YAKL test
 ##################
```

Comment:
At the bottom of this file, "set_tests_properties" function should include "IO_TEST" like:

set_tests_properties(
  DATA_TYPES_TEST
  MACHINE_ENV_TEST
  BROADCAST_TEST
  LOGGING_TEST
  IO_TEST
  YAKL_TEST
  PROPERTIES FAIL_REGULAR_EXPRESSION "FAIL"
)

Reply:
added - thanks